specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/CloudManifestFile.json

```diff
@@ -0,0 +1,53 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-06-01",
+    "title": "Azure Stack Azure Bridge Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.AzureStack/cloudmanifestfile": {
+      "get": {
+        "tags": [
+          "CloudManifestFile"
+        ],
+        "description": "Returns a cloud specific manifest JSON file.",
+        "operationId": "CloudManifestFile_Get",
+        "parameters": [
+          {
+            "$ref": "AzureStack.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CloudManifestFileResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "AzureStack.json#/definitions/ErrorResponse"
```

Comment:
Looks like this is update to existing api-version. For future version's swagger, can you update the default to use standard [schema like this](https://github.com/Azure/azure-rest-api-specs/blob/6a03b4217d5af3c3296de5cc27ab16a51481d882/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2019-12-12/HybridCompute.json#L82)?

Reply:
Will do